pkg/network/protocols/postgres/telemetry.go

```diff
@@ -0,0 +1,79 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2024-present Datadog, Inc.
+
+//go:build linux_bpf
+
+package postgres
+
+import (
+	"fmt"
+
+	libtelemetry "github.com/DataDog/datadog-agent/pkg/network/protocols/telemetry"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const numberOfBuckets = 4
+
+// Telemetry is a struct to hold the telemetry for the postgres protocol
+type Telemetry struct {
+	metricGroup *libtelemetry.MetricGroup
+
+	// exceededQueryLengthBuckets holds the counters for the different buckets of exceeding query length quires
+	exceededQueryLengthBuckets [numberOfBuckets]*libtelemetry.Counter
+	// failedTableNameExtraction holds the counter for the failed table name extraction
+	failedTableNameExtraction *libtelemetry.Counter
+	// failedOperationExtraction holds the counter for the failed operation extraction
+	failedOperationExtraction *libtelemetry.Counter
+}
+
+// createQueryLengthBuckets initializes the query length buckets
```

Comment:
let's elaborate more on the buckets, what buckets do we have? how many? what's the length of each of them?